recipes/librdkafka/all/patches/0002-fixed-lz4-find_package.patch

```diff
@@ -0,0 +1,32 @@
+--- a/packaging/cmake/Modules/FindLZ4.cmake
```

Comment:
Did you try `cmake_find_package` generator instead ?

Reply:
No, I didn't. The package is using Cmake find_package in module mode and I've patched those modules in order to use Conan libraries instead of system libraries. There's also another way to do this by changing the CMakelists.txt of the project and make it use Conan targets. Which one do you think is better?

Reply:
No, I didn't. As syoliver pointed out in one of the comments, there are some missing components in the OpenSSL recipe with cmake_find_package generator.

Reply:
I'm pretty sure those module patches are not necessary and I can remove them if I patch the CMake file of the project and use CONAN_PKG there.
Which one do you think is more appropriate? Patching the CMake file of the project or patching those modules?

Reply:
I would prefer using `generator = 'cmake', 'cmake_find_package'` only. Conan should be able to solve it without any extra cmake patch. Please, give a try with `cmake_find_package` and remove those patches, I think it will work.

Reply:
@sam-mosleh Concerning my issue with openssl you should be able to workaround it easily by replacing `rdkafka PUBLIC OpenSSL::SSL OpenSSL::Crypto`  with `rdkafka PUBLIC OpenSSL::OpenSSL` in `src/CMakeLists.txt`. It is just that I wanted it more integrated with OpenSSL ;-)

Reply:
Module patches are removed now.